Makefile

```diff
@@ -1,41 +1,66 @@
-score-compose:
+# Disable all the default make stuff
+MAKEFLAGS += --no-builtin-rules
+.SUFFIXES:
+
+## Display a list of the documented make targets
```

Comment:
The way this works is that you can add a comment above your other make targets to add them to the help output. For example:

```
## Generate a compose.yaml file from the score spec and launch it.
.PHONY: compose-up
...
```

Reply:
Gotcha, thanks, fixed.